lionblog-seongil/src/routes/SignInPage.jsx

```diff
@@ -1,5 +1,32 @@
 const SignInPage = () => {
-  return <div>Sign In Page</div>;
+  const handleSignUpSubmit = () => {
+    alert("로그인 하기"); // TODO: add api call for sign up
+  };
+  return (
+    <div className="flex flex-col items-center w-1/2">
+      <h3 className="font-bold text-2xl">회원가입</h3>
+      <form className="form gap-2" onSubmit={handleSignUpSubmit}>
+        <label htmlFor="email" className="label">
+          *이름:
+        </label>
+        <input required type="text" id="username" className="input" />
```

Comment:
이 부분에 사용자가 이메일을 입력하게 되어있는 만큼 코드에도 기능을 반영해서 일관성을 유지해 주는 건 어떨까요!!
```suggestion
          *이메일:
        </label>
        <input required type="text" id="email" className="input" />
```